src/classes/REL_Relationships_TEST.cls

```diff
@@ -310,46 +332,43 @@ public class REL_Relationships_TEST {
     public static void testAutoCreateContactInsert(){
         setupRelationshipTestData(null);
 
-        //reset the process controller to mimic a second user action
-        //Relationships_ProcessControl.hasRun = false;
-
         //insert a new Contact
-        Contact c = new Contact(LastName='ReportingIn', ReportsToId = c1.id);
+        Contact c = new Contact(LastName='ReportingIn', ReportsToId = c1.Id);
```

Comment:
HAHA, here we do it two different ways in the same parameter list :)

Reply:
Oh boy!